python/flatbuffers/builder.py

```diff
@@ -391,6 +392,14 @@ def EndVector(self, vectorNumElems):
         self.PlaceUOffsetT(vectorNumElems)
         return self.Offset()
 
+    def SerializeToString(self, s):
+        serialized = pickle.dumps(s)
```

Comment:
You're serializing a builder? what does that do?

Reply:
When we make a UnaryUnary type of call. `operation_send_message` is called. In this [line](https://github.com/grpc/grpc/blob/9a69478498232b6b42169f8a1a389b51fb4e03ec/src/python/grpcio/grpc/_channel.py#L463) `operation_send_message` is called and it has parameters `serialized_request` which is a byte string. Here is the implementation of [operation_send_message](https://github.com/grpc/grpc/blob/d31551e35faa542ad49c3f531221f3a413bf8805/src/python/grpcio/grpc/_cython/_cygrpc/records.pyx.pxi#L618) where it expects the `data` variable to be of bytes type. An exception is thrown if the `serialized_request` isn't a byte string 'Exception serializing request!'. 

The pickle module is good for serializing any python object to bytes. `pickle.dumps` returns the serialized representation of the object.

Reply:
But now you're serializing the result of one serializer + its support data structures with another serializer, which is a really bad idea. Please make sure it directly uses the serialized FlatBuffer data.

Reply:
What do you mean by "directly uses the serialized Flatbuffer data"? How would I use the Flatbuffer data without converting it into bytes?

Reply:
`request_serializer` requires a function which converts any object to bytes.

Reply:
The FlatBuffer data is already bytes. By using pickle you are wrapping it (and any data in the builder it doesn't need) a second time. This needs to be removed.